<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StaffWindowClass</class>
 <widget class="QMainWindow" name="StaffWindowClass">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>员工管理</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Cosmetic/Cosmetic.qrc">
    <normaloff>:/staff/Resources/people.png</normaloff>:/staff/Resources/people.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="100,1">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="StaffListView" name="itemView" native="true"/>
    </item>
    <item>
     <widget class="SideBar" name="sideBar" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <addaction name="actionMenu"/>
    <addaction name="actionLock"/>
    <addaction name="actionLogOff"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPersonal">
    <property name="title">
     <string>个人信息(&amp;P)</string>
    </property>
    <addaction name="actionBrowseMyInfo"/>
    <addaction name="actionBrowseMyRecords"/>
    <addaction name="actionBrowseMySalary"/>
    <addaction name="actionChangeMyPassword"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>员工设置(&amp;S)</string>
    </property>
    <addaction name="actionAddStaff"/>
    <addaction name="actionEditStaff"/>
    <addaction name="actionRemoveStaff"/>
    <addaction name="actionStaffConfig"/>
    <addaction name="actionRights"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>关于(&amp;A)</string>
    </property>
    <addaction name="actionAboutCosmetic"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>显示设置(&amp;D)</string>
    </property>
    <addaction name="actionToolBar"/>
    <addaction name="actionSmallIcon"/>
    <addaction name="actionTextLabel"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPersonal"/>
   <addaction name="menuSetting"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMenu"/>
   <addaction name="actionLock"/>
   <addaction name="separator"/>
   <addaction name="actionBrowseMyInfo"/>
   <addaction name="actionBrowseMyRecords"/>
   <addaction name="actionBrowseMySalary"/>
   <addaction name="actionChangeMyPassword"/>
   <addaction name="separator"/>
   <addaction name="actionAddStaff"/>
   <addaction name="actionRemoveStaff"/>
   <addaction name="actionEditStaff"/>
   <addaction name="actionStaffConfig"/>
   <addaction name="actionRights"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionMenu">
   <property name="icon">
    <iconset resource="../Cosmetic/Cosmetic.qrc">
     <normaloff>:/common/Resources/Downloads.png</normaloff>:/common/Resources/Downloads.png</iconset>
   </property>
   <property name="text">
    <string>主菜单(&amp;M)</string>
   </property>
   <property name="iconText">
    <string>主菜单</string>
   </property>
   <property name="toolTip">
    <string>打开主菜单</string>
   </property>
   <property name="statusTip">
    <string>打开主菜单</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionLogOff">
   <property name="icon">
    <iconset resource="../Cosmetic/Cosmetic.qrc">
     <normaloff>:/common/Resources/gnome-logout.png</normaloff>:/common/Resources/gnome-logout.png</iconset>
   </property>
   <property name="text">
    <string>注销(&amp;L)</string>
   </property>
   <property name="iconText">
    <string>注销</string>
   </property>
   <property name="toolTip">
    <string>注销当前登录用户</string>
   </property>
   <property name="statusTip">
    <string>注销当前登录用户</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../Cosmetic/Cosmetic.qrc">
     <normaloff>:/common/Resources/quit.png</normaloff>:/common/Resources/quit.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;E)</string>
   </property>
   <property name="iconText">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出科思美系统</string>
   </property>
   <property name="statusTip">
    <string>退出科思美系统</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionBrowseMyInfo">
   <property name="icon">
    <iconset resource="../Cosmetic/Cosmetic.qrc">
     <normaloff>:/common/Resources/dlgedit.png</normaloff>:/common/Resources/dlgedit.png</iconset>
   </property>
   <property name="text">
    <string>基本资料(&amp;I)</string>
   </property>
   <property name="iconText">
    <string>基本资料</string>
   </property>
   <property name="toolTip">
    <string>查看个人基本资料</string>
   </property>
   <property name="statusTip">
    <string>查看个人基本资料</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionBrowseMyRecords">
   <property name="icon">
    <iconset resource="../Cosmetic/Cosmetic.qrc">
     <normaloff>:/common/Resources/contents.png</normaloff>:/common/Resources/contents.png</iconset>
   </property>
   <property name="text">
    <string>业绩浏览(&amp;R)</string>
   </property>
   <property name="iconText">
    <string>业绩浏览</string>
   </property>
   <property name="toolTip">
    <string>查看个人业绩</string>
   </property>
   <property name="statusTip">
    <string>查看个人业绩</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionBrowseMySalary">
   <property name="icon">
    <iconset resource="../Cosmetic/Cosmetic.qrc">
     <normaloff>:/common/Resources/salary.png</normaloff>:/common/Resources/salary.png</iconset>
   </property>
   <property name="text">
    <string>工资查询(&amp;S)</string>
   </property>
   <property name="iconText">
    <string>工资查询</string>
   </property>
   <property name="toolTip">
    <string>查看工资相关信息</string>
   </property>
   <property name="statusTip">
    <string>查看工资相关信息</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionChangeMyPassword">
   <property name="icon">
    <iconset resource="../Cosmetic/Cosmetic.qrc">
     <normaloff>:/common/Resources/users.png</normaloff>:/common/Resources/users.png</iconset>
   </property>
   <property name="text">
    <string>更改密码(&amp;P)</string>
   </property>
   <property name="iconText">
    <string>更改密码</string>
   </property>
   <property name="toolTip">
    <string>更改当前登录用户密码</string>
   </property>
   <property name="statusTip">
    <string>更改当前登录用户密码</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionStaffConfig">
   <property name="icon">
    <iconset resource="../Cosmetic/Cosmetic.qrc">
     <normaloff>:/staff/Resources/people.png</normaloff>:/staff/Resources/people.png</iconset>
   </property>
   <property name="text">
    <string>属性设置(&amp;M)</string>
   </property>
   <property name="iconText">
    <string>属性设置</string>
   </property>
   <property name="toolTip">
    <string>设置员工职务、级别参数</string>
   </property>
   <property name="statusTip">
    <string>设置员工职务、级别参数</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionLock">
   <property name="icon">
    <iconset resource="../Cosmetic/Cosmetic.qrc">
     <normaloff>:/common/Resources/gpgsm.png</normaloff>:/common/Resources/gpgsm.png</iconset>
   </property>
   <property name="text">
    <string>锁定(&amp;E)</string>
   </property>
   <property name="iconText">
    <string>锁定</string>
   </property>
   <property name="toolTip">
    <string>锁定系统</string>
   </property>
   <property name="statusTip">
    <string>锁定系统</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRights">
   <property name="icon">
    <iconset resource="../Cosmetic/Cosmetic.qrc">
     <normaloff>:/common/Resources/lock.png</normaloff>:/common/Resources/lock.png</iconset>
   </property>
   <property name="text">
    <string>权限设置(&amp;P)</string>
   </property>
   <property name="iconText">
    <string>权限设置</string>
   </property>
   <property name="toolTip">
    <string>设置用户权限</string>
   </property>
   <property name="statusTip">
    <string>设置用户权限</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionAboutCosmetic">
   <property name="icon">
    <iconset resource="../Cosmetic/Cosmetic.qrc">
     <normaloff>:/common/Resources/cosmetic.png</normaloff>:/common/Resources/cosmetic.png</iconset>
   </property>
   <property name="text">
    <string>科思美(&amp;K)</string>
   </property>
   <property name="iconText">
    <string>科思美</string>
   </property>
   <property name="toolTip">
    <string>关于科思美系统</string>
   </property>
   <property name="statusTip">
    <string>关于科思美系统</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionAddStaff">
   <property name="icon">
    <iconset resource="../Cosmetic/Cosmetic.qrc">
     <normaloff>:/common/Resources/add_user.png</normaloff>:/common/Resources/add_user.png</iconset>
   </property>
   <property name="text">
    <string>添加员工(&amp;A)</string>
   </property>
   <property name="iconText">
    <string>添加员工</string>
   </property>
   <property name="toolTip">
    <string>添加新员工</string>
   </property>
   <property name="statusTip">
    <string>添加新员工</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRemoveStaff">
   <property name="icon">
    <iconset resource="../Cosmetic/Cosmetic.qrc">
     <normaloff>:/common/Resources/delete_user.png</normaloff>:/common/Resources/delete_user.png</iconset>
   </property>
   <property name="text">
    <string>删除员工(&amp;D)</string>
   </property>
   <property name="iconText">
    <string>删除员工</string>
   </property>
   <property name="toolTip">
    <string>删除列表中选定的员工</string>
   </property>
   <property name="statusTip">
    <string>删除列表中选定的员工</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionEditStaff">
   <property name="icon">
    <iconset resource="../Cosmetic/Cosmetic.qrc">
     <normaloff>:/common/Resources/edit_user.png</normaloff>:/common/Resources/edit_user.png</iconset>
   </property>
   <property name="text">
    <string>编辑员工(&amp;E)</string>
   </property>
   <property name="iconText">
    <string>编辑员工</string>
   </property>
   <property name="toolTip">
    <string>编辑列表中选定的员工</string>
   </property>
   <property name="statusTip">
    <string>编辑列表中选定的员工</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSmallIcon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>小图标</string>
   </property>
   <property name="toolTip">
    <string>工具栏显示小图标</string>
   </property>
  </action>
  <action name="actionTextLabel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>文字标签</string>
   </property>
   <property name="toolTip">
    <string>显示工具栏按钮的文字标签</string>
   </property>
  </action>
  <action name="actionToolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>工具栏</string>
   </property>
   <property name="toolTip">
    <string>显示/隐藏工具栏</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SideBar</class>
   <extends>QWidget</extends>
   <header>SideBar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StaffListView</class>
   <extends>QWidget</extends>
   <header>StaffListView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Cosmetic/Cosmetic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>menuBar</sender>
   <signal>triggered(QAction*)</signal>
   <receiver>StaffWindowClass</receiver>
   <slot>dealAction(QAction*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>9</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>dealAction(QAction*)</slot>
 </slots>
</ui>
